torch/_decomp/__init__.py

```diff
@@ -139,6 +139,8 @@ def register(op):
 def get_decompositions(
     aten_ops: Sequence[Union[OpOverload, OpOverloadPacket]],
     type: str = "post_autograd",
+    *,
+    prefer_cpp_composites: bool = False,
```

Comment:
cc @ezyang I ended up... adding this flag, lmk what you think. This issue is that @andrewor14 would like a way to set up his decomp table for `pre_dispatch`, such that the **exact same** set of decomps end up running as if you had run vanilla `make_fx()`.

When you run vanilla `make_fx()`, C++ CompositeImplicit decomps get priority over any python decomps, so this flag basically lets you explicitly ask for the same behavior.

Reply:
okey dokey